src/views/accounts/AccountChangeSecret/index.vue

```diff
@@ -45,6 +45,17 @@ export default {
   },
   computed: {
     ...mapGetters(['hasValidLicense'])
+  },
+  mounted() {
+    this.$eventBus.$on('change-tab', this.handleChangeTab)
+  },
+  beforeDestroy() {
+    this.$eventBus.$off('change-tab', this.handleChangeTab)
+  },
+  methods: {
+    handleChangeTab(tab) {
+      this.config.activeMenu = tab
+    }
   }
 }
 </script>
```

Comment:
The code appears to have been edited in between September 2021 and March 2025. Please review it again after correcting the spelling mistakes identified.